inc/Config/QueryRunner/QueryResponseParser.php

```diff
@@ -55,7 +55,8 @@ private function get_field_value( mixed $field_value, string $type_name, mixed $
 	 */
 	public function parse( mixed $data, array $schema ): mixed {
 		$json_obj = $data instanceof JsonObject ? $data : new JsonObject( $data );
-		$value = $json_obj->get( $schema['path'] ?? '$' );
+		$default_path = $schema['is_collection'] ?? false ? '$[*]' : '$';
```

Comment:
The `?? false` just reads poorly to me in this line, though it functions correctly. Does this avoid a warning or something?

We could check `isset` or `array_key_exists` or something, but that's longer.

Maybe just parens around it to make it read better?

```suggestion
		$default_path = ( $schema['is_collection'] ?? false ) ? '$[*]' : '$';
```

Reply:
Done in https://github.com/Automattic/remote-data-blocks/pull/477/commits/1f21745b46881e9036a168a419b4f15a74e0bd9d